ee/tabby-db/SQLX.md

```diff
@@ -0,0 +1,37 @@
+# Using SQLX with Tabby
+
+Tabby uses SQLX for its database, which comes with several advantages:
+
+- All queries created using the `query!()`, `query_as!()` and `query_scalar!()` macros are syntax- and schema-checked at compiletime
+- All migrations are managed by sqlx
+- A local copy of our db schema is maintained in `ee/tabby-db/schema.sqlite`
```

Comment:
could you explain a bit (or adding link) to what does this file do? why we don't need such file with rusqlite?